README.md

```diff
@@ -21,6 +21,29 @@ Optionally:
 - **Bank payments** - we provide a possibility to integrate bank payments in your app with both SCA and non-SCA options.
 - **Card payments** - we also provide card payments with a hybrid payment support, so the payment would be performed via bank if we detect that the card belongs to one of our supported banks.
 
+## Installation
+- **Swift Package Manager** - kevin. iOS SDK supports [Swift Package Manager](https://www.swift.org/package-manager/), which is the recommended option. Add a dependency to your project with: `.package(url: "https://github.com/getkevin/kevin-ios", from: "2.2.12")`
+- **CocoaPods:** - add `pod 'kevin-ios'` to your `Podfile`
+
+## Usage example
+
+### Initiate payment
+
+```
+import Kevin
+
+let paymentID = # your payment ID
+
+let configuration = try KevinPaymentSessionConfiguration.Builder(paymentId: paymentID)
+    .setPaymentType(.bank)
+    .build()
+
+KevinPaymentSession.shared.delegate = self #  Set the delegate to obtain payment  session result
```

Comment:
Also, maybe we should add at least one delegate method here to show case it?

Reply:
I wanted to keep the usage sample to a minimum but yeah, it probably makes sense to also include delegate code 👌